Header header
int32[] indices # [[lu,ru,v],...] 
